cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -1039,7 +1049,11 @@ fn aarch64_get_operands<F: Fn(VReg) -> VReg>(inst: &Inst, collector: &mut Operan
             collector.reg_def(rd);
             memarg_operands(mem, collector);
         }
-        &Inst::Pacisp { .. } => {}
+        &Inst::Pacisp { .. } => {
+            collector.reg_mod(regs::writable_link_reg());
+            collector.reg_use(regs::stack_reg());
```

Comment:
Likewise here, the instruction doesn't need any operands from RA's point of view I think.